docs/tutorials/basic/reshape_transpose.md

```diff
@@ -0,0 +1,190 @@
+
+## Difference between reshape and transpose operators
+Modyfing the shape of tensors is a very common operation in Deep Learning. For instance, when using pretrained neural networks it is often required to adjust input data dimensions to correspond to what the network has been trained on, e.g. tensors of shape `[batch_size, channels, width, height]`.  This notebook discusses briefly the difference between the operators [Reshape](http://mxnet.incubator.apache.org/test/api/python/ndarray.html#mxnet.ndarray.NDArray.reshape) and [Transpose](http://mxnet.incubator.apache.org/test/api/python/ndarray.html#mxnet.ndarray.transpose). Both allow to change the shape, however they are not the same and are commonly mistaken.
```

Comment:
Modyfing -> Modifying

often required to adjust input data dimension --> often necessary to adjust the input data dimension 

Both allow to --> Both allow you to